dts/bindings/spi/raspberrypi,pico-spi-pio.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2023 Stephen Boylan <stephen.boylan@beechwoods.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: Raspberry Pi Pico SPI via PIO
+
+compatible: "raspberrypi,pico-spi-pio"
+
+include: ["spi-controller.yaml", "raspberrypi,pico-pio-device.yaml", "reset-device.yaml"]
+
+properties:
+  clk-gpios:
+    type: phandle-array
+    required: true
+    description: |
+      Output pin for SPI clock
```

Comment:
It should better to describe that this array accept only one element.

Reply:
Hmm ... Since the description uses singular nouns, only one element is implied.  This is consistent with definition of other gpios, at least in the .yaml files for SPI, and follows the guidance in [Example phandle-arrays: GPIOs](https://docs.zephyrproject.org/latest/build/dts/phandles.html#example-phandle-arrays-gpios).  However, if you insist, I'll change it.

Reply:
I got it.
(I'm not a native English speaker, so it's easy to miss this point, sorry.)